voicemail/README.md

```diff
@@ -0,0 +1,55 @@
+# Voicemail
+
+This Function will forward incoming calls to another number during business hours and will otherwise direct to a voicemail. The voicemail will record a message and send a link of the recording via SMS to the configured phone number.
```

Comment:
It would be good to document the "business hours" this function assumes.

Additionally, did you consider making working hours configurable somehow?